assets/javascripts/discourse/connectors/user-preferences-interface/show-quick-messages.js.es6

```diff
@@ -1,7 +1,7 @@
 export default {
   shouldRender(args, component) {
-    return Discourse.SiteSettings.quick_messages_enabled &&
+    return Discourse.SiteSettings.quick_message_enabled &&
```

Comment:
@ryanerwin The rest of the PR is good, but we don't need to change this site setting name.

Reply:
I think there was a typo before... from `server.en.yml`
![image](https://user-images.githubusercontent.com/2156415/46192930-d97fb900-c326-11e8-91e1-b7d04af6af49.png)


I found this rails singular or plural cheatsheet, so I'll try remember to reference that and get with the convention... https://alexander-clark.com/blog/rails-conventions-singular-or-plural/